apps/console/src/features/users/components/bulk-import-response-list.tsx

```diff
@@ -355,8 +489,17 @@ export const BulkImportResponseList: React.FunctionComponent<BulkImportResponseL
                         disableRightActionPanel={ false }
                     >
                         <Typography variant="body2" style={ { textAlign: "right" } }>
-                            { t("console:manage.features.user.modals.bulkImportUserWizard.wizardSummary." +
-                                "totalCount") } : { totalCount }
+                            {
+                                responseOperationType === BulkImportResponseOperationTypes.USER_CREATION
+                                    ? (
+                                        t("console:manage.features.user.modals.bulkImportUserWizard.wizardSummary." +
+                                 "totalUserCreationCount") + " : " + totalUserCreationCount
```

Comment:
Let's fix indentation here. Check the other places as well.

Reply:
Resolved